aptos-move/framework/aptos-framework/sources/aptos_governance.move

```diff
@@ -31,14 +31,21 @@ module aptos_framework::aptos_governance {
     use aptos_framework::timestamp;
     use aptos_framework::voting;
 
-    /// Error codes.
+    /// The specified stake pool does not have sufficient stake to create a proposal
     const EINSUFFICIENT_PROPOSER_STAKE: u64 = 1;
+    /// This account is not the designated voter of the specified stake pool
     const ENOT_DELEGATED_VOTER: u64 = 2;
+    /// The specified stake pool does not have a sufficient remaining lockup to create a proposal or vote
```

Comment:
The specified stake pool does not have long enough remaining lockup to create a proposal or vote

Sufficient makes it seem like I don't have enough stake rather than it's too short